Makefile

```diff
@@ -278,7 +283,9 @@ TAGS_PREREQ := $(TAGS_EVIDENCE)
 endif
 
 .PHONY: generate-swagger
-generate-swagger:
+generate-swagger: $(SWAGGER_SPEC)
+
+$(SWAGGER_SPEC): $(GO_SOURCES_NO_BINDATA)
```

Comment:
Why not directly
```suggestion
generate-swagger: $(GO_SOURCES_NO_BINDATA)
```
?

Reply:
Nope.

That won't do the right thing.

Try it

Reply:
Okay, I don't have to understand why, but I accept it.

Reply:
It's to do with the dependencies - the SWAGGER_SPEC line is creating a file that acts as evidence for the build job. If all sources are older than the SWAGGER_SPEC file it doesn't need to be generated